src/TestItemRunner.jl

```diff
@@ -194,38 +194,79 @@ function run_tests(path; filter=nothing, verbose=false)
     # Run testitems
     test_setup_module = Core.eval(Main, :(module $(gensym()) end))
     test_setup_module_set = TestSetupModuleSet(test_setup_module, Set{Symbol}())
-    Test.push_testset(testset("Package"; verbose=verbose))
-    for (file, testitems) in pairs(testitems)
-        Test.push_testset(testset(relpath(file, path); verbose=verbose))
-        for testitem in testitems
-            snippets_to_run = []
-            if !isempty(testitem.option_setup)
-                for setup in testitem.option_setup
-                    key = setup
-                    if haskey(testsetups, key)
-                        testsetup = testsetups[key]
-                        if testsetup.kind==:module
-                            working_dir = dirname(file)
-                            ensure_evaled(test_setup_module_set, testsetup.filename, testsetup.code, testsetup.name, testsetup.line, testsetup.column, working_dir)
-                        elseif testsetup.kind==:snippet
-                            push!(snippets_to_run, testsetup)
+
+    @static if VERSION ≤ v"1.12"
```

Comment:
I think this comparison is not right, it would have 1.12.1 run on the new version. I think a comparison that should work would be `VERSION < v"1.13-"`.